integration/components/image/image_test.go

```diff
@@ -182,6 +186,22 @@ func TestImage(t *testing.T) {
 				fmt.Sprintf("%s/%s:%s: not found", registryAddr, testRepo, "unknown"),
 			},
 		},
+		{
+			name: "export",
+			cmd:  commonImage.NewExportCmd(),
+			args: []string{testRegistryImage, exportPath},
+			expectedStdout: []string{
+				fmt.Sprintf("Successfully exported images to %s", exportPath),
+			},
+		},
+		{
+			name: "import",
+			cmd:  commonImage.NewImportCmd(),
+			args: []string{exportPath},
+			expectedStdout: []string{
+				fmt.Sprintf("Successfully imported images:\n  %s", testRegistryImage),
```

Comment:
Is there an easy way to check if the imported image is the same as the one we exported?

Reply:
I don't think so. We could check the image digest but with the current structure of the test it's difficult to do.